apps/versions/tests.py

```diff
@@ -710,6 +716,60 @@ def test_beta_addon_beta_file(self):
         self.assert_served_locally(self.client.get(self.file_url))
 
 
+class TestDownloadsSigned(TestDownloadsBase):
+    def setUp(self):
+        super(TestDownloadsSigned, self).setUp()
+
+        # Create a fake XPI downloadable file.
+        import zipfile
+        if not os.path.exists(os.path.dirname(self.file.file_path)):
+            os.makedirs(os.path.dirname(self.file.file_path))
+
+        fp = zipfile.ZipFile(self.file.file_path, 'w')
+        fp.writestr('install.rdf', '<?xml version="1.0"?><RDF></RDF>')
+        fp.close()
+
+    def tearDown(self):
+        super(TestDownloadsSigned, self).tearDown()
+        os.unlink(self.file.signed_file_path)
+
+    @mock.patch.object(settings, 'SIGNING_SERVER_ACTIVE', True)
```

Comment:
why not use [override_settings](https://docs.djangoproject.com/en/1.7/topics/testing/tools/#django.test.override_settings)?